karate-core/src/main/java/com/intuit/karate/core/Config.java

```diff
@@ -285,6 +292,45 @@ public boolean configure(String key, Variable value) { // TODO use enum
                 return true;
             case "localAddress":
                 localAddress = value.getAsString();
+                return true;
+            case "continueOnStepFailure":
+                continueOnStepFailureMethods.clear(); // clears previous configuration - in case someone is trying to chain these and forgets resetting the previous one
+
+                boolean enableContinueOnStepFailureFeature = false;
+                Boolean continueAfterIgnoredFailure = null;
+
+                Collection<Method> selectedMethods = new HashSet<>();
+                if (value.isMap()) {
+                    Map<String, Object> map = value.getValue();
+                    List<String> stepKeywords = (List<String>) map.get("keywords");
+                    if (stepKeywords != null) {
+                        selectedMethods = StepRuntime.findMethodsByKeywords(stepKeywords);
+                    }
+
+                    continueAfterIgnoredFailure = (Boolean) map.get("continueAfter");
+                    enableContinueOnStepFailureFeature = map.get("enabled") != null && (Boolean) map.get("enabled");
+                }
+
+                if (value.isTrue() || enableContinueOnStepFailureFeature) {
+
+                    if(selectedMethods.isEmpty()) {
```

Comment:
minor code-format nit-pick, can you apply IDE format, I'd like the space after `if`